integration/integrate-with-slack.adoc

```diff
@@ -0,0 +1,24 @@
+[id="integrate-with-slack"]
+= Integrating with Slack
+include::modules/common-attributes.adoc[]
+:context: integrate-with-slack
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://slack.com/[Slack], you can forward alerts from {product-title} to Slack.
+
+High-level integration flow:
+
+. Create a new Slack App, enable Incoming Webhooks, and get a Webhook URL.
```

Comment:
s/Slack App/Slack app (app is generic and not anything product-y or special within Slack)
Same for "Incoming Webhooks" and "Webhook URL" here? Unless matching the GUI, I'd think these could be lowercase ("incoming webhooks" and "webhook URL").

These conventions would also apply throughout the rest of the PR.